examples/vnet/kubernetesvnet-azure-cni.json.env

```diff
@@ -0,0 +1 @@
+ACSE_PREDEPLOY=examples/vnet/k8s-vnet-azure-cni-predeploy.sh
```

Comment:
Note that there's no post-deploy reference here because Azure CNI custom VNET attachment doesn't need the post-deployment route table attachment step that kubenet VNET attachment requires.